src/app/pages/merge-page/merge-page.component.scss

```diff
@@ -0,0 +1,60 @@
+$primary-color: rgb(55, 111, 252);
+$dark-back-color: rgb(10, 10, 10);
+$light-back-color: rgb(121, 121, 121);
+$error-color: rgb(121, 1, 1);
+$box-shadow: 0px 2px 1px -1px rgba(0, 0, 0, 20%), 0px 1px 1px 0px rgba(0, 0, 0, 14%), 0px 1px 3px 0px rgba(0, 0, 0, 12%);
+$height: 12vh;
+
+.candidate-merge-container {
+  margin: 20px 100px;
+  padding-bottom: 10px;
+  height: auto;
+  font-family: Roboto, "Helvetica Neue", sans-serif;
```

Comment:
let's use common styles in all application

Reply:
done